addons/web/static/src/legacy/js/views/res_partner_form_controller.js

```diff
@@ -0,0 +1,43 @@
+/** @odoo-module alias=web.ResPartnerFormController **/
+
+import { _t } from 'web.core';
+import FormController from 'web.FormController';
+import ResPartnerFormViewDialog from 'web.ResPartnerFormViewDialog';
+
+
+const ResPartnerFormController = FormController.extend({
+
+    /**
+     * @override
+     */
+    _onOpenOne2ManyRecord: async function (ev) {
```

Comment:
What is the purpose of this code ? Aka: it lacks at least a docstring, as it is not crystal clear to me what it is used for :D